frame/operator/src/lib.rs

```diff
@@ -106,11 +106,11 @@ decl_module! {
 
             // remove origin operator to contracts
             <OperatorHasContracts<T>>::mutate(&operator,
-                |tree| for c in contracts.iter() { (*tree).remove(c); }
```

Comment:
What’s about using filter instead of experimental feature?

Reply:
Year, I agree.